rbac/server/db/users_query.py

```diff
@@ -56,6 +58,7 @@
                 "memberOf": fetch_relationships_by_id(
                     "role_members", user_id, "role_id", head_block_num
                 ),
+                "expired": fetch_expired_roles(user_id, head_block_num),
```

Comment:
Would we want to show expired field when the function `fetch_all_user_resources()` function is called?

Reply:
I can't imagine a use case for that at the moment.